src/pages/resources/reference/push-notifications/templates/rating.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Adobe Experience Platform SDK timer push template
+description: Timer push template supported by Adobe Campaign Classic mobile extension.
+keywords:
+- Adobe Campaign Classic
+- Push
+- Push Template
+- Rating
+---
+
+import Tabs from './tabs/rating.md'
+
+# Push template - rating
+
+A notification that allows displaying a title, text, and image, along with 3-5 image buttons that allow the user to provide "rating" feedback.
```

Comment:
in the `multi-icon` page, we did this:
```
with three to five (3-5) image buttons...
```

i suppose i don't care much which way we do it, but would ask that our docs be consistent